examples/amm/amm.py

```diff
@@ -112,7 +111,19 @@ def bootstrap(
         *,
         output: abi.Uint64,
     ):
-        """bootstraps the contract by opting into the assets and creating the pool token"""
+        """bootstraps the contract by opting into the assets and creating the pool token.
```

Comment:
@barnjamin Are you on-board with committing the generated artifacts (TEAL source + contract JSON)?

* I considered the question when seeing changes that modify contract JSON.  From a _best practices_ perspective, I imagine it's _preferable_ to commit the generated sources as a to more easily catch unintended changes.
* Additionally and out-of-scope to the PR, I can imagine building a script and/or Github Action to validate PyTeal source generates the committed artifacts.

Reply:
@barnjamin Thanks for the change in https://github.com/algorand-devrel/beaker/pull/85/commits/09cc8d1b36111d5bc5a9d721f25bf2d3ea19a9ae.  I don't have permission to resolve the thread, so let's consider the note a resolution.